pydoctor/astbuilder.py

```diff
@@ -1194,6 +1195,18 @@ def parseFile(self, path: Path) -> Optional[ast.Module]:
                 self.warning("cannot parse", str(path))
             self.ast_cache[path] = mod
             return mod
+    
+    def parseString(self, py_string:str) -> Optional[ast.Module]:
```

Comment:
Since it’s only used in the test, I’d suggest we drop the caching for parseString.